src/Nethermind/Nethermind.TxPool/TxBroadcaster.cs

```diff
@@ -97,31 +97,31 @@ internal class TxBroadcaster : IDisposable
         // only for testing reasons
         internal Transaction[] GetSnapshot() => _persistentTxs.GetSnapshot();
 
-        public void Broadcast(Transaction tx, bool isPersistent)
+        public bool Broadcast(Transaction tx, bool isPersistent)
```

Comment:
Minor cosmetic: would be good to rename method, right now we are not calling it if we want to `broadcast` (void), but when we want to `try to broadcast` (bool return depending on success). So I recommend renaming it to `TryBroadcast` and `TryStartBroadcast`

Reply:
Worth to consider before merging